### ISSUE TITLE
code_to_flag behaves weirdly with lowercase country codes

### ISSUE THREAD

**Issue by juliuste** [leodutra/country-emoji#1]
Hey 🙂 Thank you very much for providing this useful library! I noticed some unintuitive behaviour in the `code_to_flag` function, when supplying a country code in lowercase:

```rs
let correct = country_emoji::code_to_flag("DE"); // as_bytes returns [240, 159, 135, 169, 240, 159, 135, 170]
let incorrect = country_emoji::code_to_flag("de"); // as_bytes returns [240, 159, 136, 137, 240, 159, 136, 138]
```

I guess this happens because `code_to_flag`, unlike most other functions, doesn't use `.to_uppercase()` to normalize the input.

### Expected behaviour

I'd either expect all methods to return `None` for lowercase codes, or the `code_to_flag` function to return the correct flag even when supplying a lowercase country code.

I'm happy to work on a fix for the problem if you don't have the time, but I'd need to know which behaviour you'd prefer.

**Comment by leodutra** [comment]
This should be fixed now on [0.2.0](https://github.com/leodutra/country-emoji/commit/7209668ca687851aa77eb13b98398e9f7fcb8786). The next version will include minimal testing.

**Comment by leodutra** [comment]
I'll close it, for now. Feel free to reopen if you have any issues.
